tf2_eigen_kdl/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>tf2_eigen_kdl</name>
+  <version>0.15.1</version>
```

Comment:
This number needs to be the same as other packages in the repository (otherwise `catkin_prepare_release` gets cranky).  So right now this should be 0.13.8.

Reply:
Fixed in 91bebe0a9e8ab4d7dd5976c311b2ff5dfeb296f1